api/ocm/ref.go

```diff
@@ -145,14 +145,6 @@ func (r *RefSpec) Name() string {
 	return fmt.Sprintf("%s/%s//%s", r.Host, r.SubPath, r.Component)
 }
 
-func (r *RefSpec) HostPort() (string, string) {
-	i := strings.Index(r.Host, ":")
-	if i < 0 {
-		return r.Host, ""
-	}
-	return r.Host[:i], r.Host[i+1:]
-}
-
```

Comment:
Why is this being removed? As it's an API change, we should probably mention this in the release notes.

Reply:
The method has been moved to the `UniformRepositorySpec`, which is an embedded field.